core/templates/dev/head/pages/collection_editor/CollectionEditorNavbarDirective.js

```diff
@@ -20,7 +20,8 @@ oppia.directive('collectionEditorNavbar', [
   'UrlInterpolationService', function(UrlInterpolationService) {
     return {
       restrict: 'E',
-      templateUrl: 'inline/collection_editor_navbar_directive',
+      templateUrl:  UrlInterpolationService.getDirectiveTemplateUrl(
+        '/pages/collection_editor/collection_editor_navbar_directive.html'),
```

Comment:
This was done as collection_editor_navbar_directive has jinja template code in it. So, it can't be directly referenced